web/auth/guardian_serializer.ex

```diff
@@ -0,0 +1,12 @@
+defmodule CodeCorps.GuardianSerializer do
+  @behaviour Guardian.Serializer
+
+  alias CodeCorps.Repo
+  alias CodeCorps.User
+
+  def for_token(user = %User{}), do: { :ok, "User:#{user.id}" }
+  def for_token(_), do: { :error, "Unknown resource type" }
+
+  def from_token("User:" <> id), do: { :ok, Repo.get(User, id) }
```

Comment:
Just a quick comment here: `from_token/1` is invoked overtime you deserialize a JWT coming into the API, which means we're automatically incurring a round trip DB hit at the start of every call. Do we need the whole user object?